packages/components/notification/Notification.mdx

```diff
@@ -44,43 +47,43 @@ The `Notification` component can be configured in a number of different ways. He
 ```jsx
 <React.Fragment>
   <Button
-    buttonType="positive"
+    variant="positive"
     onClick={() => Notification.success('This is a success notification')}
   >
     Show success notification
   </Button>
   <Button
-    buttonType="negative"
+    variant="negative"
     onClick={() => Notification.error('This is an error notification')}
   >
     Show error notification
   </Button>
   <Button
-    buttonType="warning"
+    variant="secondary"
     onClick={() => Notification.warning('This is a warning notification')}
   >
     Show warning notification
   </Button>
 </React.Fragment>
 ```
```

Comment:
Kind of a nitpicky suggestion

To give some space between the buttons, you could do something like what we have in the Buttons README now
https://github.com/contentful/forma-36/blob/master/packages/forma-36-react-components/src/components/Button/README.mdx

basically wrap them with `<Grid rowGap="spacingXs" rows="repeat(4, auto)" justifyContent="start">`